macros/src/main/scala/gestalt/macros/Monadless.scala

```diff
@@ -41,17 +44,23 @@ object Monadless {
 
 object Transformer {
```

Comment:
It's basically a refactoring, but the macros become more robust, not just some crappy hack.